ksp_plugin_test/plugin_compatibility_test.cpp

```diff
@@ -274,6 +278,86 @@ TEST_F(PluginCompatibilityTest, Reach) {
 }
 #endif
 
+TEST_F(PluginCompatibilityTest, DISABLED_Butcher) {
+  StringLogSink log_warning(google::WARNING);
+  not_null<std::unique_ptr<Plugin const>> plugin = ReadPluginFromFile(
+      R"(P:\Public Mockingbird\Principia\Saves\1119\1119.proto.b64)",
+      /*compressor=*/"gipfeli",
+      /*decoder=*/"base64");
+  // TODO(phl): Check that we mention a compatibility path here once something
+  // changes.
+  EXPECT_THAT(log_warning.string(),
+              Not(HasSubstr("pre-Gröbner")));
+  auto const& orbiter =
+      *plugin->GetVessel("e180ca12-492f-45bf-a194-4c5255aec8a0");
+  EXPECT_THAT(orbiter.name(), Eq("Mercury Orbiter 1"));
+  auto const begin = orbiter.psychohistory().begin();
+  EXPECT_THAT(begin->time,
+              Eq("1966-05-10T00:14:03"_TT + 0.0879862308502197 * Second));
+  EXPECT_THAT(begin->degrees_of_freedom,
+              Eq(DegreesOfFreedom<Barycentric>(
+                  Barycentric::origin + Displacement<Barycentric>(
+                                            {-9.83735958466250000e+10 * Metre,
+                                             -1.05659916408781250e+11 * Metre,
+                                             -4.58171358797500000e+10 * Metre}),
+                  Velocity<Barycentric>(
+                      {+2.18567382812500000e+04 * (Metre / Second),
+                       -1.76616533203125000e+04 * (Metre / Second),
+                       -7.76112133789062500e+03 * (Metre / Second)}))));
+
+  auto const& mercury = plugin->GetCelestial(2);
+  EXPECT_THAT(mercury.body()->name(), Eq("Mercury"));
+
+  plugin->RequestReanimation(begin->time);
+
+  while (mercury.trajectory().t_min() > begin->time) {
```

Comment:
Polling?!  Please use `WaitForReanimation`.

Reply:
There is no WaitForReanimation in the plugin.